language-support/ts/daml-react/hooks.ts

```diff
@@ -131,6 +138,7 @@ export function useFetchByKey<T extends object, K, I extends string>(template: T
  * @ignore
  */
 export const useExercise = <T extends object, C, R>(choice: Choice<T, C, R>): (cid: ContractId<T>, argument: C) => Promise<R> => {
+  console.log('useExercise is deprecated. Please use "useLedger" instead.');
```

Comment:
Is there a `@deprecated` pragma we could add to the doc comment?

Reply:
so right now, the docs for this are not shown at all because of the `@ignore` pragma. we can also use the `@deprecate` pragma to show the docs as deprecated.

Reply:
changed it to @deprecated.